src/v2/Apps/Partner/Routes/Artists/index.tsx

```diff
@@ -23,13 +22,10 @@ export const ArtistsRoute: React.FC<ArtistsRouteProps> = ({
   partner,
   match,
 }) => {
-  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
+  const navBarHeight = useNavBarHeight()
 
   const handleArtistClick = () => {
-    const offset =
-      PARTHER_NAV_BAR_HEIGHT +
-      (isMobile ? MOBILE_NAV_HEIGHT : NAV_BAR_HEIGHT) +
-      20
+    const offset = PARTHER_NAV_BAR_HEIGHT + navBarHeight + 20
```

Comment:
Note the typo in the const `PARTHER_NAV_BAR_HEIGHT`

Reply:
Fixed